docs/database/database-spans.md

```diff
@@ -478,6 +479,37 @@ Semantic conventions for individual database systems or specialized instrumentat
 MAY specify a different `db.query.summary` format as long as produced summary remains
 relatively short and its cardinality remains low comparing to the `db.query.text`.
 
+## Propagating context to databases
+
+**Status**: [Development][DocumentStatus]
+
+Instrumentations SHOULD propagate the context information to the SQL queries following [sqlcommenter](https://google.github.io/sqlcommenter/spec/).
+
+| Attribute      | Type   | Description                           | Require level     | Stability                                                      |
```

Comment:
Maybe we can change this table and name it something like "recommended propagators"?

Reply:
Added header `Recommended attributes` to make it more intuitive.